lib/presentation/widget/shared_space_details/shared_space_details_widget.dart

```diff
@@ -0,0 +1,217 @@
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        ListTile(
+          leading: Padding(
+              padding: EdgeInsets.only(left: 10),
+              child: SvgPicture.asset(imagePath.icSharedSpace,
+                  color: AppColor.primaryColor, width: 20, height: 24, fit: BoxFit.fill)),
+          title: Text(details.sharedSpaceNodeNested.name,
+              style: TextStyle(
+                  color: AppColor.activeDarkGrayColor,
+                  fontWeight: FontWeight.normal,
+                  fontSize: 14.0)),
+          trailing: Text(
+            filesize(details.quota.usedSpace.size),
+            style: TextStyle(
+              fontSize: 14,
+              fontStyle: FontStyle.italic,
+              fontWeight: FontWeight.normal,
+              color: AppColor.uploadFileFileSizeTextColor,
+            ),
+          ),
+        ),
+        Divider(),
+        Column(
+          children: [
+            _sharedSpaceInformationTile(AppLocalizations.of(context).modified,
+                formatter.format(details.sharedSpaceNodeNested.modificationDate.toLocal())),
+            _sharedSpaceInformationTile(AppLocalizations.of(context).created,
+                formatter.format(details.sharedSpaceNodeNested.creationDate.toLocal())),
+            _sharedSpaceInformationTile(
+                AppLocalizations.of(context).my_rights,
+                toBeginningOfSentenceCase(details.sharedSpaceNodeNested.sharedSpaceRole.name
+                    .toString()
+                    .split('.')
+                    .last
+                    .toLowerCase())),
+            _sharedSpaceInformationTile(AppLocalizations.of(context).max_file_size,
+                filesize(details.quota.maxFileSize.size)),
+          ],
+        ),
+        Divider(),
+        Container(
+            margin: EdgeInsets.only(left: 24, top: 10),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(AppLocalizations.of(context).description,
+                    style: TextStyle(
+                        fontWeight: FontWeight.normal,
+                        fontSize: 16,
+                        color: AppColor.activeDarkGrayColor)),
+                Padding(
+                    padding: EdgeInsets.only(top: 18),
+                    child: Text('tes tes teste stse',
+                        style: TextStyle(
+                            color: AppColor.inactiveGrayColor,
+                            fontStyle: FontStyle.italic,
+                            fontWeight: FontWeight.normal,
+                            fontSize: 16.0)))
+              ],
+            ))
+      ],
+    );
+  }
+
+  ListTile _sharedSpaceInformationTile(String category, String value) {
+    return ListTile(
+      dense: true,
+      title: Text(category,
+          style: TextStyle(
+              color: AppColor.inactiveGrayColor,
```

Comment:
I think this color must be `#FFACACAC`

Reply:
![image](https://user-images.githubusercontent.com/48354990/112619791-02199880-8e28-11eb-94c2-b5fe6418b67a.png)